conf/zoo_sample.cfg

```diff
@@ -6,6 +6,9 @@ initLimit=10
 # The number of ticks that can pass between 
 # sending a request and getting an acknowledgement
 syncLimit=5
+# The number of ticks that followers can take
+# to connect to the leader after leader election
+connectToLeaderLimit=5
```

Comment:
I think we probably want to rename this variable to something like `connectToLearnerMasterLimit`, or if we keep the current name, at least document that this will also impact the socket timeout between observer and learner master. This is unfortunate, so maybe we ultimately want to rename the method of `connectToLeader` to something like `connectToLearnerMaster`?

Reply:
Naming is the hardest part...

Reply:
I will update the pull request to reflect the suggested changes. I will be renaming connectToLeaderLimit to connectToLearnerMasterLimit